api/src/main/java/org/pmiops/workbench/interceptors/AuthInterceptor.java

```diff
@@ -39,8 +39,8 @@
 /**
  * Intercepts all non-OPTIONS API requests to ensure they have an appropriate auth token.
  *
- * <p>Checks handler methods for annotations
- * like @AuthorityRequired({Authority.REVIEW_RESEARCH_PURPOSE}) to enforce granular permissions.
```

Comment:
Do we need to update the Authority enum as well?

Reply:
yes I think we can remove it - no users have this Authority in any environment